core-blocks/gallery/gallery-image.js

```diff
@@ -115,7 +82,13 @@ class GalleryImage extends Component {
 		// Disable reason: Each block can be selected by clicking on it and we should keep the same saved markup
 		/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/onclick-has-role, jsx-a11y/click-events-have-key-events */
 		return (
-			<figure className={ className } tabIndex="-1" onKeyDown={ this.onKeyDown } ref={ this.bindContainer }>
+			<figure
+				className={ className }
+				tabIndex="-1"
+				onClick={ this.props.onSelect }
```

Comment:
Drive-by review, but does the change from `onFocus` in the child `RichText ` component to the `onClick` here present any accessibility issues? How does keyboard navigation work after this change (I didn't see it mentioned in the manual testing being done)?

Reply:
I haven't tested it with the keyboard, I will double check if it should be updated to `onFocus`.

Reply:
Updated to use `onFocus`. It looks like Gallery block isn't navigable with keyboard :(